cli/pcluster/config/validators.py

```diff
@@ -572,6 +572,24 @@ def url_validator(param_key, param_value, pcluster_config):
     return errors, warnings
 
 
+def s3_bucket_validator(param_key, param_value, pcluster_config):
+    errors = []
+    warnings = []
+
+    if urlparse(param_value).scheme == "s3":
+        try:
+            bucket = param_value.split("/")[2]
+            boto3.client("s3").head_bucket(Bucket=bucket)
+        except ClientError:
+            warnings.append("The S3 bucket '{0}' does not exist or you do not have access to it.".format(param_value))
```

Comment:
What about adding a message like: _"Please be sure the cluster nodes have access to it."_?

Reply:
Fixed by https://github.com/aws/aws-parallelcluster/pull/1613